### DISCUSSION TITLE
Students package in github

### DISCUSSION THREAD

**Discussion by shamnad-sherief** [github/india#105] [Students]
Is there any package available for students as it would help to grow their carrier?

**Comment by rohitnb** [comment]
@shamnad-sherief - We have the Student Developer pack with access to tons of amazing developer tools. Check out the Student Dev pack [here](https://education.github.com/pack).

**Reply by shamnad-sherief** [reply]
thanku. got it

**Reply by shamnad-sherief** [reply]
Submitted my details with ID card. Still there is no response from github. They say having an edu-mail make easy to confirm. As u know no university in India provide edu-mail for students.What to do?

**Reply by Surbhdey** [reply]
@shamnad-sherief - Hope you got the access to Student developer pack by now. And if you are a third-year or pre-final year student then apply for **GitHub Externship Program**. See-through the guidelines and T&C's

Please apply using the [link](https://github-externships.github.io/externship/index.html).

**Reply by varghesejose2020** [reply]
where can see my student developer pack expiry date

**Reply by TarunavBA** [reply]
When will I get access to my GitHub student pack?

**Reply by varghesejose2020** [reply]
https://education.github.com/pack/offers @TarunavBA

**Reply by scmishra-cse** [reply]
> @shamnad-sherief - Hope you got the access to Student developer pack by now. And if you are a third-year or pre-final year student then apply for **GitHub Externship Program**. See-through the guidelines and T&C's
> 
> Please apply using the [link](https://github-externships.github.io/externship/index.html).

Registrations are closed for now when will it re-open again.
